weed/server/filer_server_handlers_write_autochunk.go

```diff
@@ -374,6 +375,12 @@ func SaveAmzMetaData(r *http.Request, existing map[string][]byte, isReplace bool
 		}
 	}
 
+	//acp
+	acp := r.Header.Get(s3_constants.ExtAcpKey)
+	if len(acp) > 0 {
+		metadata[s3_constants.ExtAcpKey] = []byte(acp)
+	}
+
```

Comment:
Only `ExtAcpKey` is processed here, `ExtOwnershipKey` is through the Ownership related rest api